File: easy/longest_word/longest_word.py
import sys

f = open(sys.argv[1])
lines = f.read().strip().splitlines()

for line in lines:
    line = line.split(' ')   
    longest = 0
    max_word_length = 0
    for x in range(len(line)):
        if len(line[x]) > max_word_length:
            longest = x
            max_word_length = len(line[x])

    print(line[longest])

File: easy/longest_word/longest_word.js
var fs = require('fs')
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line !== "") {
        var longestWord = ""
        line.split(' ').map(function (word) {
            if (word.length > longestWord.length) {
                longestWord = word;
            }
        });
        console.log(longestWord);
    }
});
